solvers/prox_newton.py

```diff
@@ -10,7 +10,7 @@ class Solver(BaseSolver):
 
     stopping_strategy = "iteration"
 
-    install_cmd = "conda"
+    install_cmd = 'conda'
```

Comment:
I think black prefers " to ' so let's go revert to " if possible. Anyways, such changes are generally avoidable